src/index.js

```diff
@@ -150,8 +155,101 @@ const renderLevel = (root, levelNumber) => {
         .join("")}
     </tr>
   `;
+
+  renderSettings = () => {  
+    const settingsHTML = `
+    <div id="modal">
+      <div class="modal-content">
+        <span class="close material-symbols-outlined">cancel</span>
+        <h1>Setting</h1>
+      </div>
+    </div>
+    `;
+    document.querySelector("#overlay-screen").innerHTML = settingsHTML;
+
+    document.querySelector(".close").addEventListener("click", () => {
+      document.querySelector("#modal").style.display = "none";
+    });
+  };
+
+  renderHelp = () => {  
+    const helpHTML = `
+    <div id="modal">
+      <div class="modal-content">
+        <span class="close material-symbols-outlined">cancel</span>
+        <h1>How To Play</h1>
```

Comment:
The `<h1>` here feels a bit overwhelmingly large. Maybe `<h2>`, since it's not as grand and important as the `<h1>` for our title screen. We could style the menu headers to be a notch or two more subtle than the main title.